app/services/api/v3/actors/basic_attributes.rb

```diff
@@ -180,7 +180,7 @@ def importer_summary
 
         def summary_of_total_trade_volume(profile_type)
           if @trade_total_current_year_raw.zero?
-            return "<span class=\"notranslate\">#{@node.name.humanize}</span> \
+            return "<span class=\"notranslate\">#{@node.name.upcase}</span> \
```

Comment:
I think there are 3 more instances of `@node.name.humanize` in this file that should be changed in the same way. Quickest way to test this in development is to delete the computed summaries in psql:
`rails dbconsole`
`> UPDATE nodes_with_flows SET actor_basic_attributes = NULL;`
Longer version would be to recompute the summaries in development, rather than delete them:
`rails console`
`> Api::V3::RefreshProfiles.new.call`

In sandbox / production best to use the background jobs version:
`RAILS_ENV=sandbox bundle exec rake basic_attributes:refresh_later`

Reply:
done, I think I made it work locally so we would run this when merged right? 

```
RAILS_ENV=sandbox bundle exec rake basic_attributes:refresh_later
```

Reply:
after it's deployed, that needs to be run on `sandbox-data`